plugins/log_request_plugin/src/main.rs

```diff
@@ -11,18 +11,13 @@ use smartrest::{
 };
 
 const AGENT_LOG_DIR: &str = "/var/log/tedge/agent";
-const MQTT_SESSION_NAME: &str = "log plugin mqtt session";
-const HTTP_SESSION_NAME: &str = "log plugin http session";
 
-/// creates an mqtt client with a given `session_name`
-pub async fn create_mqtt_client(
-    session_name: &str,
-) -> Result<mqtt_channel::Connection, anyhow::Error> {
+/// creates an mqtt client
+pub async fn create_mqtt_client() -> Result<mqtt_channel::Connection, anyhow::Error> {
     let tedge_config = get_tedge_config()?;
     let mqtt_port = tedge_config.query(MqttPortSetting)?.into();
     let mqtt_config = mqtt_channel::Config::default()
         .with_port(mqtt_port)
-        .with_session_name(session_name)
         .with_subscriptions(mqtt_channel::TopicFilter::new_unchecked(
             C8yTopic::SmartRestResponse.as_str(),
         ));
```

Comment:
This is okay but could be improved.

* There is no need to subscribe, since no messages are expected.
* The session should be clean, since we are not using the same name from one session to another. In fact this should be implictly done by the `mqtt_channel` crate when no name is provided for the session.

Reply:
I created a PR so it's useless to explicitly set the clean session flag when no name is provided. See https://github.com/thin-edge/thin-edge.io/pull/917